### PR TITLE
Working redirects nov 2017

### PR DESCRIPTION
**Pull request by leungant**

### CODE REVIEW

**Comment by leungant** [comment]
projects_list view does not exist yet, so urls.py is set to redirect to user_page as the first page instead.

**Comment by marizoldi** [comment]
Oh I just did it yesterday under Arun's PR. I basically subclassed success url from ActivationView. Have a look at my last commit in the PR ActivationComplete. Although I would prefer it to first go to activation_complete and then redirect as I think you did it. Another thing is that the user page is the /home not the projects. The projects will be presented using D3 in the /home which is the main dashboard.

**Comment by leungant** [comment]
How unfortunate, thought that message on Slack was to take a look at this. Also I thought /projects was to be the main home view, which was why this was implemented.

At least now all the sources etc from the original pull request are all done! Closing pull request and deleting changes. Sorry to waste your time!

**Comment by marizoldi** [comment]
Hey, may be worth to implement the first activaiton_complete.html and then redirection. Could you close this and add this part only to Arun's ActivationComplete PR as the next commit (since we started doing this in that branch)?

File: erehwon/erehwon/templates/registration/activation_complete.html [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<meta http-equiv="refresh" content="1; url=/projects" />
```
Comment (marizoldi) [inline erehwon/erehwon/templates/registration/activation_complete.html]: this is what we need, no?!

Comment (marizoldi) [reply erehwon/erehwon/templates/registration/activation_complete.html]: redirecting to /home

**Comment by marizoldi** [comment]
Added implementation to ActivationComplete PR.

**Pull request closed without merging**